lib/connection.js

```diff
@@ -92,6 +92,10 @@ connection.prototype.end = function(data) {
     return promiseCallback.apply(this.connection, ['end', arguments]);
 };
 
+connection.prototype.release = function(data) {
```

Comment:
This makes it apply to all connections. `release` doesn't exist on non-pool connections so it shouldn't be defined.